UltraStar Play/ProjectSettings/ProjectSettings.asset

```diff
@@ -149,7 +149,8 @@ PlayerSettings:
   resolutionScalingMode: 0
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
-  applicationIdentifier: {}
+  applicationIdentifier:
+    iOS: org.ultrastardx.play.alpha
```

Comment:
I guess it is only ultrastar instead of ultrastardx, according to the games name. Also, is it the usual thing to do to add the version in the app identifier? Wouldn't that break any possible updating?

Reply:
For me at least, and only with iOS, yes. (For Android it's not that big of a deal)

It's the usual thing to not use the final app identifier before going to production.
And it's correct that it will break updating. But that's by design. That way the production app won't have to deal with old data from non-released version.

And although we won't have multiple supported versions (there will be only 1 master)
Its better to do pre-release build with a non-final bundle id.

And that's because of Apple.
When you build an iOS app that bundle-id is linked to your Apple account. Meaning your account will be the only account that can every build with that bundle id. 
I figured it would not be wise to link the final the bundle id to my account.

The final bundle id should be linked to your Apple account, as maintainer of this repo.